### PR TITLE
OSDOCS-4048: Adding User Telemetry Information

### PR DESCRIPTION
**Pull request by EricPonvelle**
Manual cherry-pick of #56971

### CODE REVIEW

**Comment by mburke5678** [comment]
@EricPonvelle This one bothers me. The difference in the number of files changed and the +/- numbers are way off. Is that expected.

**Comment by EricPonvelle** [comment]
@mburke5678 I think it's fixed now

File: rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc [hunk 1]
```diff
@@ -46,6 +46,5 @@ include::modules/rosa-sts-creating-a-cluster-quickly-cli.adoc[leveloffset=+1]
...
 * For details about using the `auto` and `manual` modes to create the required STS resources, see xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc#rosa-understanding-deployment-modes_rosa-sts-creating-a-cluster-with-customizations[Understanding the auto and manual deployment modes].
 * For more information about using OpenID Connect (OIDC) identity providers in AWS IAM, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html[Creating OpenID Connect (OIDC) identity providers] in the AWS documentation.
-* For more information about troubleshooting ROSA cluster installations, see xref:../rosa_support/rosa-troubleshooting-installations.adoc#rosa-troubleshooting-installations[Troubleshooting installations].
-* For steps to contact Red Hat Support for assistance, see xref:../rosa_architecture/rosa-getting-support.adoc#rosa-getting-support[Getting support for Red Hat OpenShift Service on AWS].
-
+* For more information about troubleshooting ROSA cluster installations, see xref:../sd_support/rosa-troubleshooting-installations.adoc#rosa-troubleshooting-installations[Troubleshooting installations].
```
Comment (mburke5678) [inline rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc:49]: Should [this link](https://github.com/openshift/openshift-docs/pull/56971/files#diff-70f029dfbd5101198efe7b42502bffe71bdfb40a41bc06981dafbaf5c6af04d9R54) be here also?

**Comment by mburke5678** [comment]
@EricPonvelle Much better! One question. Otherwise LGTM

File: rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc (hunk 1 above)
Comment (EricPonvelle) [reply rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc:49]: @mburke5678 Looking at that link to the Troubleshooting installations, it seems fine and should be included.

Comment (mburke5678) [reply rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc:49]: @EricPonvelle Sorry. I thought you missed one that is in the original PR. Not sure what I was looking at.

**Pull request merged**